format/spec.md

```diff
@@ -1453,13 +1457,15 @@ Each sort field in the fields list is stored as an object with the following pro
 
 | V1       | V2       | V3       | Field            | JSON representation | Example     |
 |----------|----------|----------|------------------|---------------------|-------------|
-| required | required | required | **`transform`**  | `JSON string`       | `bucket[4]` |
-| required | required | omitted  | **`source-id`**  | `JSON int`          | 1           |
+| required | required | optional | **`transform`**  | `JSON string`       | `bucket[4]` |
```

Comment:
Transform is still required, right? This looks like a typo and you meant to change `source-ids` to optional?

Reply:
Yes, I was one-off, thanks 🙌